ocs_ci/helpers/disruption_helpers.py

```diff
@@ -22,9 +23,24 @@ class Disruptions:
     resource_count = 0
     selector = None
     daemon_pid = None
+    cluster_kubeconfig = ""
 
     def set_resource(self, resource, leader_type="provisioner"):
         self.resource = resource
+        if (config.ENV_DATA["platform"] in constants.MANAGED_SERVICE_PLATFORMS) and (
+            resource in ["mds", "mon", "mgr", "osd"]
```

Comment:
Maybe we can add a constant or a var with the ceph pods list `["mds", "mon", "mgr", "osd"]`

Reply:
Created a constant in disruption_helpers.py. Not adding in constants.py because these particular names of the pods are useful in disruption_helpers only.